lib/index.js

```diff
@@ -14,7 +14,7 @@ const {
 } = require("./config-array-factory");
 
 const { CascadingConfigArrayFactory } = require("./cascading-config-array-factory");
-const { ModuleResolver } = require("./shared/relative-module-resolver");
+const moduleResolver = require("./shared/relative-module-resolver");
```

Comment:
```suggestion
const { resolve }= require("./shared/relative-module-resolver");
```

this should be `resolve`, the same name as exported